java-checks/src/test/files/checks/DisallowedClassCheck.java

```diff
@@ -1,3 +1,9 @@
+import unknown;
```

Comment:
cannot comment unchanged line inside github, but could you please add a comment why line 36 is Compliant? ... is has String as well and it's not intuitive for me
`((java.util.Optional<String>) o).map(value -> 1);`

Reply:
I added the comment: // Compliant, ignore inferred type
This explain why "value" is compliant, but I didn't challenge why type cast is compliant.